packages/legend-query-builder/src/stores/shared/ValueSpecificationEditorHelper.ts

```diff
@@ -134,19 +136,27 @@ export const buildDefaultInstanceValue = (
   graph: PureModel,
   type: Type,
   observerContext: ObserverContext,
+  enableInitializingDefaultValue: boolean,
 ): ValueSpecification => {
   const path = type.path;
   switch (path) {
     case PRIMITIVE_TYPE.STRING:
-    case PRIMITIVE_TYPE.BOOLEAN:
     case PRIMITIVE_TYPE.STRICTDATE:
     case PRIMITIVE_TYPE.DATETIME:
     case PRIMITIVE_TYPE.NUMBER:
     case PRIMITIVE_TYPE.DECIMAL:
     case PRIMITIVE_TYPE.FLOAT:
```

Comment:
where are `case PRIMITIVE_TYPE.BINARY` and `case PRIMITIVE_TYPE.BYTE`?

Reply:
I talked with @MauricioUyaguari about it and we decided to remove it, because it's not actually supported in constants and filters.
- If you try to DND a byte property into the filter panel, you get an error because no operations are registered for the byte type.
- If you create a binary or byte constant and then go to "Edit Query in Pure," you also get an error because it's unable to build the query.
Since it's not actually supported, we decided to remove it, and also removed binary and byte from the constant type selection dropdown, since no one should be using it.

Reply:
https://github.com/finos/legend-studio/pull/2333
Could you please confirm this PR won't be broken?

Reply:
It looks like the same actions still work:
![ByteEditor](https://github.com/finos/legend-studio/assets/9127428/c2ed5376-507d-400e-890d-ea230dd194ef)

I think the only places where I would have concerns are the `EqualToAsssertionEditor` component in `packages/legend-application-studio/src/components/editor/editor-group/testable/TestableSharedComponents.tsx` and the `createFunctionTest` function in `packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/function-activator/testable/FunctionTestableState.ts`. Do you know if that component/function would ever get used with a byte or binary type?

Reply:
May I know if there is a lot of extra work you need to do for adding these two back?
Besides this one, your PR LGTM.

Reply:
I think it shouldn't be an issue to add these two back, especially since we removed them from the constants type dropdown. I'll work on adding them back.

Reply:
I have re-added BYTE and BINARY to the function switch statement.